src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private final double firstLeg;

```

Comment:
These names tell nothing about the sides they are referring to. Use:
- topSize
- bottomSide
- height

Reply:
Not fixed